RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -5958,3 +5958,69 @@ int meatReserve()
 	
 	return reserve_gnasir + reserve_diary + reserve_island + reserve_extra;
 }
+
+// This function returns the number of bander runaways that will be available after maximizing for fam weight
+int auto_banderRunsAvailable() {
+	if (in_quantumTerrarium()) {
+		if ((my_familiar() == $familiar[Frumious Bandersnatch] && have_skill($skill[The Ode to Booze])) || my_familiar() == $familiar[Pair of Stomping Boots]) {
+			maximize("familiar weight, -tie", false); // Need to maximize or simulate maximizing in accordance with standard autoscend procedure
+			return (((familiar_weight (my_familiar()) + weight_adjustment()) / 5) - get_property("_banderRunaways").to_int());
+		}
+		else {
+			return 0;
+		}
+	}
+	else if (have_familiar($familiar[Frumious Bandersnatch]) && have_skill($skill[The Ode to Booze])) {
+		use_familiar($familiar[Frumious Bandersnatch]);
+		maximize("familiar weight, -tie", false);
+		return (((familiar_weight ($familiar[Frumious Bandersnatch]) + weight_adjustment()) / 5) - get_property("_banderRunaways").to_int());
+	}
+	else if (have_familiar($familiar[Pair of Stomping Boots])) {
+		use_familiar($familiar[Pair of Stomping Boots]);
+		maximize("familiar weight, -tie", false);
+		return (((familiar_weight ($familiar[Pair of Stomping Boots]) + weight_adjustment()) / 5) - get_property("_banderRunaways").to_int());
+	}
```

Comment:
this can likely all be rolled into the previous else if as it looks like a copy/paste

Reply:
I removed some of the horrible stuff that this part of the function was doing. I toyed with incorporating it into the previous else if statement but, and I'm admittedly nothing resembling experienced or an expert, I didn't see a good way to do it. I'd love further suggestions on the newer structure though, I'm loving every opportunity to learn here.